templates/fullwidth.html

```diff
@@ -3,10 +3,6 @@
 
 {% block body_class %}tpl-fullwidth{% endblock %}
 
-{% block extend_base %}
-<div class="container">
```

Comment:
AFAIK only protected files get overridden on an update currently. It has been tested beforehand on the xplorer theme